editor/store/actions.js

```diff
@@ -353,9 +353,18 @@ export function editPost( edits ) {
 	};
 }
 
-export function savePost() {
+/**
+ * Returns an action object to save the post.
+ *
+ * @param {Object} options          Options for the save.
+ * @param {Object} options.autosave Perform an autosave if true.
```

Comment:
`{Object}` should be `{boolean}`

Reply:
fixed in https://github.com/WordPress/gutenberg/pull/6257/commits/dbd3ab661dddef2398a9b2c77d03f58b633af3a2